### PR TITLE
fixes issue #132 by taking x&y scrollposition into account

### PR DESCRIPTION
**Pull request by meierjan**
Fixes the scrolling bug in #132.

The problem was that scrolling-positons were not taken into account when `canScrollUp` tries to find the touched element. 

Therefore I am 'descrolling' the intersection-hitbox.

This picture tries to illustrate it:

img_2563

(of course -scrollY on y-axis not -scrollX)

I think the failed travis build is due to a too old version.
> Plugin is too old, please update to a more recent version, or set ANDROID_DAILY_OVERRIDE environment variable to "c6930c23d36ab6b6b38444329718db02cc97d20d"

### CODE REVIEW

**Comment by markrietveld** [comment]
@meierjan Sorry it took so long to get back to you here. We updated the build tools. If you rebase this on master here it should pass checks again.

File: bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java [hunk 1]
```diff
@@ -450,11 +450,13 @@ private boolean canScrollUp(View view, float x, float y) {
             ViewGroup vg = (ViewGroup) view;
             for (int i = 0; i < vg.getChildCount(); i++) {
                 View child = vg.getChildAt(i);
-                int childLeft = child.getLeft();
-                int childTop = child.getTop();
-                int childRight = child.getRight();
-                int childBottom = child.getBottom();
-                boolean intersects = x > childLeft && x < childRight && y > childTop && y < childBottom;
+
+                int childLeft = child.getLeft() - view.getScrollX();
```
Comment (markrietveld) [inline bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java]: These changes make sense to me.

File: bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java [hunk 2]
```diff
@@ -450,11 +450,13 @@ private boolean canScrollUp(View view, float x, float y) {
...
+                int childTop = child.getTop() - view.getScrollY();
+                int childRight = child.getRight() - view.getScrollX();
+                int childBottom = child.getBottom() - view.getScrollY();
+
+                boolean intersects = childLeft < x && x < childRight && childTop < y && y < childBottom;
```
Comment (markrietveld) [inline bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java]: Please undo the changes here, and undo the added newlines. They're not functional.

Comment (meierjan) [inline bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java]: What do you mean by `Please undo the changes here` ?

File: bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java (hunk 1 above)
Comment (markrietveld) [inline bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java]: Y

File: bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java (hunk 2 above)
Comment (markrietveld) [inline bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java]: You changed `x > childLeft` to `childLeft < x` for example.

Comment (meierjan) [inline bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java]: Okay, I will change that even though I consider it more readable. But I guess this is just to match your style guides. Thanks for the heads up

**Comment by meierjan** [comment]
I did what you asked me for in the comments. Thanks for the review.

**Comment by markrietveld** [comment]
Thanks for contributing!
